packages/components/src/utils/colors-values.js

```diff
@@ -169,6 +170,7 @@ export const COLORS = {
 	darkOpacity: DARK_OPACITY,
 	darkOpacityLight: DARK_OPACITY_LIGHT,
 	mediumGray: G2.mediumGray,
+	gray: G2.gray,
```

Comment:
Also not sure if exposing gray in this way is desirable or if these are meant to be aliased with names.